### PR TITLE
Start a "Using BookWyrm" section

### PR DESCRIPTION
**Pull request by zacharyzollman**
- add closing parenthesis and comma to style_guide.md
- add posting_statuses.md

This is not at all comprehensive, just thought it would be nice to start on this.

### CODE REVIEW

File: content/using-bookwyrm/posting-statuses.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Title: Posting statuses
+Date: 2022-04-30
+Order: 1
+
+Posting on BookWyrm is similar to [posting on Mastodon](https://docs.joinmastodon.org/user/posting/) in that statuses have visibility settings, content warnings, and mentions. However, polls and attachments are not supported.
+
+## Types of posts
```
Comment (mouse-reeve) [inline content/using-bookwyrm/posting-statuses.md:7]: I would refine this slightly - there are three ways to post _about a book_. It's probably not useful info for this page, but technically there are also replies and generated posts, so while I think this is the right framing and scope, I think just specifying that these are how to post about a book narrows it down?

File: content/contributing/style_guide.md [hunk 2]
```diff
@@ -49,5 +49,5 @@ Some particular things that Bookwyrm contributors have found useful to remember
 
 ### Buttons and Links
 
-* Use a `<button>` element for anything that exists to trigger a JavaScript action (e.g. hiding or unhiding a form) or sends a `POST` request (e.g. submitting a form
-* Use an `<a>` element for anything that triggers a `GET` request. Usually, an anchor (`<a>`) element should not be styled as a button (`class="button"`), though there are some exceptions such as "Cancel" buttons. If in doubt, ask for advice in your pull request
+* Use a `<button>` element for anything that exists to trigger a JavaScript action (e.g. hiding or unhiding a form) or sends a `POST` request (e.g. submitting a form)
```
Comment (mouse-reeve) [inline content/contributing/style_guide.md:52]: Woops thank you!

**Review by mouse-reeve — CHANGES_REQUESTED** [review]
Thank you for starting this! This addresses a real gap in the documentation. Extremely minor feedback bellow 😄

File: content/using-bookwyrm/posting-statuses.md (hunk 1 above)
Comment (zacharyzollman) [reply content/using-bookwyrm/posting-statuses.md]: that makes sense! I updated it so it doesn't leave that out anymore

**Comment by mouse-reeve** [comment]
Thank you so much for this, I think this is going to grow to be a super helpful section of the documentation!

**Pull request merged**